### DISCUSSION TITLE
Rad Pro: Radex support

### DISCUSSION THREAD

**Discussion by Quartno** [Gissio/radpro#99] [Q&A] [answered]
Hello ! (Sorry for the copy paste was at first on the wrong comment section)

I have a Radex 1503+ for 6 years now. I kinda like this counter and add more factures can maybe transform this piece in a masterpiece.
I've recently disasembled this thing and I can tell you he is a good option for a firmware update:
1 extra empty Geiger counter tube place.

So, I tried to look a little more Closer and I have seen 2 pins alone. I think they are there for the first injection of the original firmware and if i'm right, there is a posibility to make a adaptator and connect the Radex on a pc or anything and then, intalling rad pro on it (maybe a required manipulation like holding the power button to be detected as a USB Key or something like that...)

There is anyone who knows how to do that ?

Thanks by advance and thanks to creator who made radpro firmware.

**Comment by Gissio** [comment, marked as answer]
Rad Pro is designed for STM32 32-bit processors and requires approximately 36 kB flash memory. The Radex 1503+ has an 8-bit AT89C4051-24 processor with 4 kB flash memory.

It's like trying to make a camel go through the eye of a needle.
